### PR TITLE
Beam tutorial page in documentation

### PR DESCRIPTION
**Pull request by dimaopen**
Added  sf-light scenario: created input for beam, where all modes are set (coming from activitySim). TAZ file from sfbay scenario is used. Downsample to have 5k households.
Written tutorial.rst.

- - -
This change is [Reviewable](https://reviewable.io/reviews/LBNL-UCB-STI/beam/3819)

### CODE REVIEW

File: docs/inputs.rst [hunk 1]
```diff
@@ -404,6 +406,10 @@ One can add multiple different RH fleets into the array **beam.agentsim.agents.r
 * initialization.procedural.initialLocation.home.radiusInMeters: radius within which the initial location is taken
 * initialization.filePath: this file is loaded when initialization.initType is "FILE"
 * initialization.parking.filePath: parking zones defined for ridehail fleet; it may be empty.
+* stopFilePath: an optional file that contains ride-hail stop coordinates. If this file is set then the ride-hail vehicles
+  cat pick up/drop off passengers only on the given stops. For a small fraction of passengers a wrong sequence of events
```
Comment (nikolayilyin) [inline docs/inputs.rst]: a typo - 'cat' -> 'can'

File: docs/inputs.rst [hunk 2]
```diff
@@ -404,6 +406,10 @@ One can add multiple different RH fleets into the array **beam.agentsim.agents.r
...
+  can be produced (i.e. a person can reach the pickup stop after they enter the ride-hail vehicle).
```
Comment (nikolayilyin) [inline docs/inputs.rst]: Maybe it would be better to mention that it is not that wrong sequence - only time of events (and order) is messed up?

**Comment by dimaopen** [comment]
Test!

File: docs/tutorial.rst [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+Urbansim scenario usually contains all the leg modes defined. That means that the agents don't make a mode choice and
+just follow the assigned mode. In order to make agents do the mode choice we can clear the modes. Find
+*beam.urbansim.fractionOfModesToClear* section in the file *test/input/sf-light/sf-light-urbansim-5k-hh.conf* and
+set *allModes* parameter to 1.0::
+
+    beam.urbansim.fractionOfModesToClear {
+      allModes = 1.0
+
+This means that all the modes defined in the scenario should be cleared.
```
Comment (nikolayilyin) [inline docs/tutorial.rst]: This means that all the modes defined in the scenario **will** be cleared.

**Review by nikolayilyin — APPROVED** [review]
LGTM (apart from few nitpicks)

**Comment by dimaopen** [comment]
Test!

**Pull request merged**